apps/twap/src/app/utils/computeOrderUid.ts

```diff
@@ -0,0 +1,28 @@
+import type { Order } from '@cowprotocol/contracts';
+import {
+  OrderParameters,
+  OrderSigningUtils,
+  SupportedChainId,
+} from '@cowprotocol/cow-sdk';
+import { returnNever } from '../types/order';
+
+function isOrder(order: Order | OrderParameters): order is Order {
+  return true;
+}
+
+export async function computeOrderUid(
+  chainId: SupportedChainId,
+  owner: string,
+  order: Order | OrderParameters
+): Promise<string> {
+  const { computeOrderUid: _computeOrderUid } = await import(
```

Comment:
Since you don't care about bundle size on backend, you can make this method synchronous

Reply:
Good point, moved it